content/webapp/components/SearchFilters/SearchFilters.Desktop.tsx

```diff
@@ -87,6 +93,20 @@ const SearchFiltersDesktop: FunctionComponent<SearchFiltersSharedProps> = ({
             hasNoResults={hasNoResults}
           />
         </FilterDropdownsContainer>
+
+        {booleanFilters?.map(f => (
+          <Space
+            key={f.id}
+            $v={{ size: 'm', properties: ['margin-bottom'] }}
+            style={{ height: '32px', display: 'flex', alignItems: 'center' }}
```

Comment:
This is to make it match the dropdown button heights. Not a mega fan of setting a height though, so happy to discuss solutions.

Reply:
do the dropdown button heights change height at different viewport widths (as a result of responsive type)?

Reply:
I don't think so, and a quick test seemed to confirm it; they all disappear from small tablet/big mobile format to make way for the big FILTERS button instead